gpu/segmentation/include/pcl/gpu/segmentation/impl/gpu_extract_clusters.hpp

```diff
@@ -41,6 +41,26 @@
 
 #include <pcl/gpu/segmentation/gpu_extract_clusters.h>
 
+#include <cuda_runtime_api.h>
+#include <cuda.h>
+
+pcl::Indices economical_download(const pcl::gpu::NeighborIndices& indices,
+        pcl::Indices& sizes, int max_answers) {
+    indices.sizes.download(sizes);
+    pcl::Indices results, tmp;
```

Comment:
`tmp` HAS to be of type `std::vector<int>` due to L55

Reply:
That makes sense - thank you!